Resources/config/schema/seo-1.0.xsd

```diff
@@ -18,10 +18,9 @@
         <xsd:attribute name="title" type="xsd:string" />
         <xsd:attribute name="description" type="xsd:string" />
         <xsd:attribute name="original-route-pattern" type="original_route_pattern" />
-        <xsd:attribute name="content-key" type="xsd:string" />
```

Comment:
if we still support the old way in the Configuration class, we should also support it in the schema.